### PR TITLE
CDM-26: Create Category Element and Grid components

### PR DESCRIPTION
**Pull request by danielaprado**
### Description

- Create Category Element component
- Create Category Grid component
- Desktop & Mobile responsive

### Attachments
- **Desktop**
image
- **Mobile**
image
image

### Reviewers

- [x] @catarinapm 
- [x] @mariasbl

### CODE REVIEW

**Review by mariasbl — APPROVED** [review]

File: fe/app/assets/icons/RoundArrow.tsx
Comment (catarinapm) [inline fe/app/assets/icons/RoundArrow.tsx:1]: No need to create this icon! Use instead our component RoundArrowButton 

image

File: fe/app/components/CategoryElement/CategoryElement.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          <div className="max-md:hidden">
+            <div
+              className={clsx(
+                "flex flex-col justify-center items-center md:gap-4 gap-3"
+              )}
+            >
+              <p className="text-center text-xs text-white line-clamp-4">
+                {description}
+              </p>
+              <RoundArrow className="h-7" />
```
Comment (catarinapm) [inline fe/app/components/CategoryElement/CategoryElement.tsx:52]: Use our component <RoundArrowButton/>

File: fe/app/components/CategoryElement/CategoryElement.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          <div className="md:hidden">
+            <div
+              className={clsx(
+                "flex flex-col justify-center items-center md:gap-6 gap-3"
+              )}
+            >
+              <p className="text-center md:text-sm text-xs text-white line-clamp-4">
+                {description}
+              </p>
+              <RoundArrow className="h-5" />
```
Comment (catarinapm) [inline fe/app/components/CategoryElement/CategoryElement.tsx:66]: Same here!

File: fe/app/components/CategoryGrid/CategoryGrid.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        {rows.map((category) => (
+          <CategoryElement category={category} isDisabled={hasEnteredGrid} />
+        ))}
+      </div>
+      <div className="flex max-md:flex-col">
+        {cols.map((category) => (
+          <CategoryElement
+            category={category}
+            isDisabled={hasEnteredGrid}
+            description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet."
```
Comment (catarinapm) [inline fe/app/components/CategoryGrid/CategoryGrid.tsx:28]: Create a description prop!

File: fe/i18n/messages/pt.json [hunk 4]
```diff
@@ -1,4 +1,26 @@
 {
+  "general": {
+    "CategoryElement": {
+      "category": {
+        "edition": { "key": "edition", "value": "Edições" },
+        "masterclass": { "key": "masterclass", "value": "Masterclasses" },
+        "workshop": { "key": "workshop", "value": "Workshops" },
+        "creative-talks": {
+          "key": "creative-talks",
```
Comment (catarinapm) [inline fe/i18n/messages/pt.json:9]: Why do we have key and value? Since the key is the same as the property name

**Review by catarinapm — CHANGES_REQUESTED** [review]
check out the comments I've made!

File: fe/app/components/CategoryGrid/CategoryGrid.tsx (hunk 3 above)
Comment (danielaprado) [reply fe/app/components/CategoryGrid/CategoryGrid.tsx:28]: Confused 😕  that is not a description prop?

File: fe/i18n/messages/pt.json (hunk 4 above)
Comment (danielaprado) [reply fe/i18n/messages/pt.json:9]: Only on English, on Portuguese it will be different 😄

**Review by catarinapm — APPROVED** [review]

**Pull request merged**